posix.mak

```diff
@@ -519,7 +520,10 @@ test:
 # Changelog generation
 ################################################################################
 
-pending_changelog:
-	@echo "This command will be available soon."
+pending_changelog_impl: ${STABLE_DMD}
+	$(STABLE_RDMD) $(TOOLS_DIR)/changed.d -o changelog/pending.dd "v${LATEST}..upstream/stable"
```

Comment:
FYI: if you only want to display the manual changes and hide the changes from resolved Bugzilla issues, we can just remove the `"v${LATEST}..upstream/stable"` part.
Querying Bugzilla takes about 10-15s.